pkg/internal/checkup/executor/executor.go

```diff
@@ -116,12 +119,15 @@ func (e Executor) Execute(ctx context.Context, vmiName, podName, podContainerNam
 	if err != nil {
 		return status.Results{}, err
 	}
-
 	var trafficGeneratorSrcPortStats portStats
 	trafficGeneratorSrcPortStats, err = trexClient.GetPortStats(ctx, trafficSourcePort)
 	if err != nil {
 		return status.Results{}, err
 	}
+	_, err = trexClient.GetPortStats(ctx, trafficDestPort)
+	if err != nil {
+		return status.Results{}, err
+	}
```

Comment:
nit: Please add newlines before and after this block of code.

Reply:
DONE